targets/unity-v2/make.js

```diff
@@ -484,7 +484,7 @@ function getCustomApiLogic(tabbing, apiCall)
 {
     if (apiCall.name === "ExecuteFunction")
         return tabbing + "var localApiServerString = PlayFabSettings.LocalApiServer;\n" +
-            tabbing + "if (!string.IsNullOrEmpty(localApiServerString) && PlayFab.Internal.PlayFabUtil.ShouldDebugFunction(request.FunctionName))\n" +
+            tabbing + "if (!string.IsNullOrEmpty(localApiServerString))\n" +
```

Comment:
I'm still confused at to why changes under targets/unity-v2 are showing up in a PR for Unreal.

Reply:
Please look at changes with respect to master instead of changes to previous PR when evaluating this.
Unity local debugging is already checked in; this branch contains those changes as it branched off of master post GDC which contained local debugging changes needed to SDKGenerator. 
Having clicked "Files Changes" on the PR, and Ctrl+F'd for "unity" I do not see anything come back = no changes in this PR to unity SDK.